jest.config.ts

```diff
@@ -0,0 +1,16 @@
+import type { JestConfigWithTsJest } from "ts-jest";
+
+const jestConfig: JestConfigWithTsJest = {
+  preset: "./typescript/esm/jest-preset.js",
+  extensionsToTreatAsEsm: [".ts"],
+  transform: {
+    "^.+\\.tsx?$": [
+      "ts-jest",
+      {
+        useESM: true,
+      },
+    ],
+  },
+};
+
+export default jestConfig;
```

Comment:
![Question](https://img.shields.io/badge/review-question-yellow)

Is this config used for the actual testing? I find you remove the `defaultESM` at jest-preset.ts then I'm wondering if underlying ESM support could work or not

Reply:
I have re-added `...defaultESM` 👍